core/test-runtime/src/lib.rs

```diff
@@ -248,6 +262,22 @@ impl GetRuntimeBlockType for Runtime {
 	type RuntimeBlock = Block;
 }
 
+/// Adds one to the given input and returns the final result.
+#[inline(never)]
+fn benchmark_add_one(i: u64) -> u64 {
+	i + 1
+}
+
+/// A `Cell` that is tagged as sync, as we know that it is not modified by multiple threads.
```

Comment:
why is that again? we even have an issue filed for some kinds of limited-capacity multithreading.